!Command
positional: []
named: []
command:
- fc_graph_to_utgs
parent:
subcommands: []
help_flag:
usage_flag:
version_flag:
help_text: "Traceback (most recent call last):\n  File \"/tmp/tmpdoouzo4r/bin/fc_graph_to_utgs\"\
  , line 11, in <module>\n    load_entry_point('falcon-kit==0.4.0', 'console_scripts',\
  \ 'fc_graph_to_utgs')()\n  File \"/tmp/tmpdoouzo4r/lib/python2.7/site-packages/falcon_kit/mains/graph_to_utgs.py\"\
  , line 46, in main\n    G_asm = AsmGraph(\"sg_edges_list\", \"utg_data\", \"ctg_paths\"\
  )\n  File \"/tmp/tmpdoouzo4r/lib/python2.7/site-packages/falcon_kit/fc_asm_graph.py\"\
  , line 59, in __init__\n    self.load_sg_data(sg_file)\n  File \"/tmp/tmpdoouzo4r/lib/python2.7/site-packages/falcon_kit/fc_asm_graph.py\"\
  , line 67, in load_sg_data\n    with open(sg_file) as f:\nIOError: [Errno 2] No\
  \ such file or directory: 'sg_edges_list'\n"
generated_using:
- --help
